.travis.yml

```diff
@@ -2,20 +2,12 @@ language: go
 sudo: false
 
 go:
-  - 1.9.x
-  - 1.10.x
```

Comment:
I think that shouldn't delete against 1.9/1.0 testing because users might still be use older go version.

Reply:
Question for `use older go version.`
Old go can't use swag binary? or You say use case is contributor write swag code?

Reply:
user still can work if they still use go 1.9/1.10 version.

Reply:
I understand.